scripts/devel/test-perlin.lua

```diff
@@ -1,4 +1,13 @@
 -- Generates an image using multiple octaves of perlin noise.
+--[[=begin
+
+devel/test-perlin
+=================
+Generates an image using multiple octaves of perlin noise.
+
+Used by `3dveins`?
```

Comment:
3dveins does use this type of RNG, but not this script, so I don't think it's worth mentioning here.